src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,638 @@
+                if(isNil(option) || isEmpty(option.displayName)) {
+                  return null;
+                }
+                return option;
+              }
+            },
+          }),
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
+            id: 'itDirector',
+            description: 'I certify that the individual listed below is my IT Director',
+            placeholder: 'IT Director',
+            title: '1.7 Information Technology (IT) Director',
+            validators: [FormValidators.REQUIRED],
+            ariaLevel: ariaLevel + 1,
+            onChange: ({key: name, value,}) => formFieldChange({name, value}),
+            defaultValue: itDirector
+          })
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h3(['1.8 Cloud Use Statement*']),
+          span([
+            'Will you perform all of your data storage and analysis for this project on the ',
+            a({
+              rel: 'noopener noreferrer',
+              href: 'https://anvil.terra.bio/',
+              target: '_blank'
+            }, ['AnVIL']),
+            '?'
+          ]),
+          div([cloudRadioGroup])
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          div({className: 'row no-margin'}, [
+            div({
+              isRendered: !anvilUse && anvilUse !== '',
+              className: 'computing-use-container',
+              style: {
+                backgroundColor: showValidationMessages && isCloudUseInvalid ? 'rgba(243, 73, 73, 0.19)' : 'inherit'
+              }
+            }, [
+              div({className: 'row no-margin'}, [
+                div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
+                  h(FormField, {
+                    id: 'cloudUse',
+                    disabled: !isNil(darCode),
+                    validators: [FormValidators.REQUIRED],
+                    type: FormFieldTypes.CHECKBOX,
+                    toggleText: 'I am requesting permission to use cloud computing to carry out the research described in my Research Use Statement',
+                    checked: cloudUse,
+                    onChange: ({ key: name, value }) => formFieldChange({name, value})
+                  })
+                ])
+              ]),
+              div({className: 'row no-margin'}, [
+                div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
+                  h(FormField, {
+                    id: 'localUse',
+                    disabled: !isNil(darCode),
+                    validators: [FormValidators.REQUIRED],
+                    type: FormFieldTypes.CHECKBOX,
+                    toggleText: 'I am requesting permission to use local computing to carry out the research described in my Research Use Statement',
+                    checked: localUse,
+                    onChange: ({ key: name, value }) => formFieldChange({name, value})
+                  })
```

Comment:
are threse mutually exclusive? i.e. should these be in a radio group or is it possible to need both local and cloud computing?

Reply:
I think it could be possible to need both! This is the logic from the original request page, so I'd like to keep it the same unless otherwise stated, but this section was also not shown in the mockups

Reply:
You can select both. A researcher could theoretically handle some computations locally and more complex calculations on cloud based systems, so the form should facilitate selecting both. The text fields for more info should only apply to cloud computing since it's a third party service relative to us (Broad) and the researcher's institution.

Reply:
You can select both. More often than not researchers/labs will leverage third-party services to handle more complex and/or niche operations.